### PR TITLE
Add Flutter examples for feature flag integrations

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What does this PR do?

Added examples for feature flag evaluation for Flutter.

Also fix a few language definitions for iOS / Android code snippets.

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 1]
```diff
@@ -134,11 +134,20 @@ Each time a feature flag is evaluated, add the following function to send the fe
 
 Each time a feature flag is evaluated, add the following function to send the feature flag information to RUM:
 
-   ```javascript
+   ```kotlin
    GlobalRum.get().addFeatureFlagEvaluation(key, value);
    ```
 
 {{% /tab %}}
+{{% tab "Flutter"}}
```
Comment (drichards-87) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:142]:
```suggestion
{{% tab "Flutter" %}}
```

File: content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md [hunk 2]
```diff
@@ -134,11 +134,20 @@ Each time a feature flag is evaluated, add the following function to send the fe
...
+
+Each time a feature flag is evaluated, add the following function to send the feature flag information to RUM:
+
+   ```dart
+   DatadogSdk.instance.rum?.addFeatureFlagEvaluation(key, value);
+   ```
+
+{{% /tab }}
```
Comment (drichards-87) [inline content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md:150]:
```suggestion
{{% /tab %}}
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Hi @fuzzybinary. The build job for this PR is failing. I believe it's because of a missing % in one of the tab shortcodes. Could you fix the tabs (i.e., commit my suggestions), and then I'll review the rest of the PR. Thanks!

**Review by drichards-87 — APPROVED** [review]
Thanks for updating the PR. It's approved now. Let me know when it's ready to be merged.

**Comment by fuzzybinary** [comment]
@drichards-87 Ready now. This was documenting already available APIs.

**Pull request merged**